examples/direct_fidelity_estimation.py

```diff
@@ -263,6 +263,37 @@ def _estimate_pauli_traces_general(qubits: List[cirq.Qid],
     return pauli_traces
 
 
+def _sample_pauli_traces(pauli_traces, n_trials, n_clifford_trials):
```

Comment:
type annotations including return type annotation (for my edification!)

Reply:
Done.